verification/src/main.rs

```diff
@@ -1,8 +1,10 @@
-use verification::{run_http_server, Config};
+use clap::Parser;
+use verification::{run_http_server, Args, Config};
 
 #[tokio::main]
 async fn main() -> std::io::Result<()> {
     env_logger::init();
-    let config = Config::parse().expect("Failed to parse config");
+    let args = Args::parse();
```

Comment:
okay, but I will add `default()` instead of `new()`
<img width="733" alt="изображение" src="https://user-images.githubusercontent.com/41516657/175512366-51a224ab-0634-44f5-a68f-96c37e744df5.png">